packages/medusa/src/loaders/helpers/subscribers/__tests__/index.spec.ts

```diff
@@ -0,0 +1,114 @@
+import { MedusaContainer } from "@medusajs/types"
+import { join } from "path"
+import { containerMock, eventBusServiceMock } from "../__mocks__"
+import { SubscriberLoader } from "../index"
+
+describe("SubscriberLoader", () => {
+  const rootDir = join(__dirname, "../__fixtures__", "subscribers")
+
+  const pluginOptions = {
+    important_data: {
+      enabled: true,
+    },
+  }
+
+  let registeredPaths: string[] = []
+
+  beforeAll(async () => {
+    jest.clearAllMocks()
+
+    const paths = await new SubscriberLoader(
+      rootDir,
+      containerMock as unknown as MedusaContainer,
+      pluginOptions
+    ).load()
+
+    if (paths) {
+      registeredPaths = [...registeredPaths, ...paths]
+    }
+  })
+
+  it("should register each subscriber in the '/subscribers' folder", async () => {
+    // As '/subscribers' contains 3 subscribers, we expect the number of registered paths to be 3
+    expect(registeredPaths.length).toEqual(3)
+  })
+
+  it.skip("should have registered subscribers for 5 events", async () => {
```

Comment:
Why are some of these tests skipped?

Reply:
@olivermrbl no idea TBH, I just took what was there originally, let me try enabling them.

Reply:
@olivermrbl it needed a small fix, Fixed it and enabled the tests

Reply:
Great